auth/src/test/java/com/firebase/ui/auth/viewmodel/GenericIdpAnonymousUpgradeLinkingHandlerTest.java

```diff
@@ -152,4 +152,13 @@ private void setupAnonymousUpgrade() {
         when(mMockUser.isAnonymous()).thenReturn(true);
         when(mMockAuth.getCurrentUser()).thenReturn(mMockUser);
     }
+
+    private void mockOAuthProvider(String providerId) {
+        // TODO(samstern): I wish we did not have to do this but the OAuthProvider() builder
+        //                 throws a NPE and we can't fix it due to b/172544960
```

Comment:
I'll get to that later today, sorry it's causing you trouble

Reply:
Thanks for getting to this!